src/cubehelix.jl

```diff
@@ -0,0 +1,45 @@
+
+# following
+#  https://www.mrao.cam.ac.uk/~dag/CUBEHELIX/
+# original source in f77
+#  https://www.mrao.cam.ac.uk/~dag/CUBEHELIX/cubhlx.f
+# 
+# published in 
+#  Green, D. A., 2011, `A colour scheme for the display of astronomical intensity images', Bulletin of the Astronomical Society of India, 39, 289.
+#    (2011BASI...39..289G at ADS.) 
+
+import Color: RGB
+
+# parameters: N number of entries in result colormap (btw: Array(RGB{Float64},N))
+#  rots: rotations -> added term
+#  start: startpoint in H -> 0.0 -> Red
+#  hue: saturation of hue -> 0.0 -> grayscale only, >1.0 -> clipping in some areas
+#  gamma: gamma factor 
+
+function cubehelix(N::Int,rots::Real,start::Real,hue::Real,gamma::Real)
```

Comment:
Equivalent to your defaults, you can just do:
```
function cubehelix(N::Integer,rots::Real=0.5,start::Real=0,hue::Real=1,gamma::Real=1)
```
However, I would tend to use keywords in this situation: positional arguments get confusing when there are so many arguments, and the function-call overhead (which is worse with keywords) is not an issue here.  i.e.
```
function cubehelix(N::Integer; rots::Real=0.5,start::Real=0,hue::Real=1,gamma::Real=1)
```
(note the semicolon).   Is there a more descriptive variable name than `rots`?